service/src/test/java/org/sunbird/learner/actors/coursemanagement/CourseBatchManagementActorTest.java

```diff
@@ -369,4 +384,126 @@ public void testUpdateCompletedCourseBatchFailureWithStartDateAndEndDate() {
             .getCode()
             .equals(ResponseCode.invalidBatchStartDateError.getErrorCode()));
   }
+
+  @Test
+  public void testCreateCourseBatchInviteOnlySuccess() {
+    mockCreateRequest(true);
```

Comment:
Like performUpdateCourseBatchSuccessTest and performUpdateCourseBatchFailureTest, define performCreateCourseBatchSuccessTest and performCreateCourseBatchFailureTest.